boards/arm/lc823450/lc823450-xgevk/configs/bt/defconfig

```diff
@@ -93,8 +93,6 @@ CONFIG_NET_ROUTE=y
 CONFIG_NET_STATISTICS=y
 CONFIG_NET_TCP=y
 CONFIG_NET_TCP_KEEPALIVE=y
-CONFIG_NET_TUN=y
-CONFIG_NET_TUN_PKTSIZE=1500
```

Comment:
@jerpelea 

Could you revert this change and add `CONFIG_ALLOW_BSD_COMPONENTS=y` instead?

Reply:
By default we should not enable other licenses to the default configs except the sim defconfigs.
Each developer can enable the BSD components.

Reply:
OK.